### ISSUE TITLE
[Filter Panel]: QA: Status indicator after each filter selection and after also clearing them is not announced by screen reader

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9392]
### Description

Page Element:
Sub options present in the following filter options:
Technologies
Business needs

Issue Description:
Status indicator after each filter selection and after clearing them

Issue Description:
"following status changes are not conveyed for the screen reader user instentaniously. 
The number displayed before the clear button after each selection indicating the number of filters selected. And the same disappears after the activation of ""clear"" button."

Expected:
1.	When user makes any selection in the filter, screen reader is expected to announce, the total number of items added in the Filter such as "Total number of items in the filter is 4 (or "Added to the filter 4”
2.	When user activates clear button, Screen reader is expected to announce, “Your selection has been cleared” which would help the user to know.

User Impact:
If the screen reader fails to identify a dynamic content change, it affects users who are blind as they may not know about the change in the page.

### Component(s) impacted

Filter Panel

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.38.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component Staging- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
Closing in favor of #10589
